components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -270,7 +270,8 @@ def _unwrapunits(self, obj, units=None):
             return obj
         return obj.getValue()
 
-    def _getQueryString(self, opts=None):
+    @classmethod
+    def _getQueryString(klass, opts=None):
```

Comment:
The first argument to a class method is a reference to the class. As `class` is a "reserved" word in Python so we have to use something. `klass` is a classical style that we inherited from Java. `cls` is also fine and in line with the PEP8 suggestions as @joshmoore has already mentioned.